ddionrails/instruments/models/images.py

```diff
@@ -80,3 +83,25 @@ def save(
             using=using,
             update_fields=update_fields,
         )
+
+    def store_image(self, file: BinaryIO, path: Union[List, str]):
+        """ Store an image for this QuestionImage """
+        django_file = File(file)
+
+        _folder = self._create_folder_structure(path)
+        _filer_image, _ = Image.objects.get_or_create(folder=_folder, file=django_file)
+        _filer_image.save()
```

Comment:
You do you call _filer_image.save() explicitly here?
Is there some custom save() method on the  _filer_image?